pkg/webhook/admission/servingruntime/servingruntime_webhook.go

```diff
@@ -170,3 +185,40 @@ func validateServingRuntimePriority(newSpec *v1alpha1.ServingRuntimeSpec, existi
 	}
 	return nil
 }
+
+func validateMultiNodeVariables(newSpec *v1alpha1.ServingRuntimeSpec) error {
```

Comment:
don't we have a method that does it already?

Reply:
I already explained this with https://github.com/kserve/kserve/pull/3972#discussion_r1792154052